aptos/example/mint_nft/app/src/App.tsx

```diff
@@ -0,0 +1,153 @@
+import React from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import { PriceFeed } from "@pythnetwork/pyth-common-js";
+import { AptosClient } from "aptos";
+import { AptosPriceServiceConnection } from "@pythnetwork/pyth-aptos-js";
+
```

Comment:
Could we put a big comment here along the lines of "Please read https://docs.pyth.network/consume-data before building with Pyth."